spec/fixtures/mixed_default_settings/manifests/init.pp

```diff
@@ -0,0 +1,13 @@
+class mixed_default_settings {
```

Comment:
I'm not sure what `mixed_default_settings` means in this context?

Reply:
mixes default value_name and non-default valuename

`'hklm\Software\foo\\'` vs `'hklm\Software\foo'`

Reply:
I'll add comments into the test to explain this.

Reply:
Add an ASCII art reg tree to show what's going on